tests/framework/utils/snapshot.go

```diff
@@ -72,6 +83,23 @@ func (k8sh *K8sHelper) snapshotController(action string) error {
 	return nil
 }
 
+func (k8sh *K8sHelper) getManifestFromURL(url string) (string, error) {
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to get snapshotter controller yaml")
```

Comment:
```suggestion
              return "", errors.Wrap(err, "failed to get snapshot controller yaml")
```

Reply:
Actually I'm going to remove "snapshot" from these error messages since the method may be used in the future for other manifests as well.